src/features/blockchain/config/models.ts

```diff
@@ -124,11 +110,53 @@ export const useExecuteOperation = (
 
       throw new Error('Invalid operation type');
     },
-    [registry, walletStore],
-  );
-
-  return {
-    executeOperation,
-    isOperationValidated,
+    isOperationValidated: true,
   };
 };
+
+export const defaultOperations: ModelOperations = {
+  executeOperation: async () => {
+    throw new Error('Operations not supported');
+  },
+  isOperationValidated: false,
+};
+
+export const blockchainModels: Record<string, BlockchainModelConfig> = {
+  'gpt-4o': {
+    name: 'gpt-4o',
+    description:
+      'A robust and powerful model for executing blockchain-specific actions',
+    tools: messageRegistry.getToolDefinitions(),
+    systemPrompt: messageRegistry.getSystemPrompt(),
+    introText: messageRegistry.getIntroText(),
+    components: {
+      transaction: {
```

Comment:
what those components here and how do they differ from the ones defined in the `EvmTransferMessage`'s inProgressComponent